### DISCUSSION TITLE
Where is the winner 'fil'?

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9110] [General Q&A] [answered]
“Total Blocks:1
Total Reward:20.5104 FIL“
There is no record in the wallet. Where did it go?

**Comment by stuberman** [comment, marked as answer]
25% of the block rewards goes into the miner actor as **available**
75% of the block reward goes into the miner actor as **vesting** (over the course of 6 months)

To see the reward run `lotus-miner info` and look at **Miner Balance**

**Reply by stuberman** [reply]
Any available balance can be withdrawn into the Owner Wallet using `lotus-miner actor withdraw --help`
